pkg/testcase/privateregistry.go

```diff
@@ -0,0 +1,108 @@
+package testcase
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/rancher/distros-test-framework/pkg/customflag"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/gomega"
+)
+
+var token string
+
+func TestBuildAirgapCluster(cluster *shared.Cluster) {
+	Expect(cluster.Status).To(Equal("cluster created"))
+	Expect(cluster.ServerIPs).ShouldNot(BeEmpty())
+
+	if cluster.BastionConfig.PublicIPv4Addr != "" {
+		shared.LogLevel("info", "Bastion Public IP: "+cluster.BastionConfig.PublicIPv4Addr)
+		shared.LogLevel("info", "Bastion Public DNS: "+cluster.BastionConfig.PublicDNS)
+	}
+	shared.LogLevel("info", "Server Node IPs: %v", cluster.ServerIPs)
```

Comment:
I would remove this as well as cluster.go covers this

Reply:
https://github.com/rancher/distros-test-framework/pull/130/files#r1775892710